src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final int COLOR_COUNT = 10;
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int colorNumber = random.nextInt(COLOR_COUNT);
+
+        switch (colorNumber) {
+            case 1:
+                return "Black";
```

Comment:
we can have less colors, but let's create a separate `enum Color` and take values from it, without switch

Reply:
created enum Color instead of switch case